cmd/run.go

```diff
@@ -56,7 +56,7 @@ func Run(version, commit, date string) {
 			"valid_types":                 "7z, br, bz2, gz, lz4, rar, sz, tar, tgz, xz, zip, zst, zz",
 			"default_type":                "",                          // default is empty, but needs to be set to avoid kong error
 			"default_max_extraction_size": strconv.Itoa(1 << (10 * 3)), // 1GB
-			"default_max_files":           strconv.Itoa(1000),          // 1000 files
+			"default_max_files":           strconv.Itoa(100000),        // 100k files
```

Comment:
Is it possible to reference the constants here so you only need to maintain the value in one place?

Reply:
Hehe, great ask, but I moved away from exposing default values of the package. The cli is mainly just a nice to heave, so I focus primarily to have clean package and no frontend specific exposure. Furthermore, that should also reduce the efforts in case we need to refactor.